ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml

```diff
@@ -1,78 +1,72 @@
-  debug:
-    msg:
-    - "user.info: "
-    - "user.info: CCN Roadshow Dev Track provisioned for {{ num_users }} user(s)"
+  agnosticd_user_info:
+    msg: |
+      CCN Roadshow Dev Track provisioned for {{ num_users }} user(s)
 
 - name: output workshop info guides
-  debug:
-    msg:
-    - "user.info: "
-    - "user.info: URL for attendees to get a username assigned to them and links to labs:"
-    - "user.info: "
-    - "user.info: https://get-a-username-labs-infra.{{ route_subdomain }}"
-    - "user.info: "
-    - "user.info: You should share this URL (or a shortlink for it) -- It is all they will need to get started!"
-    - "user.info: "
-    - "user.info: [Instructor Only] To access the admin see which user is assigned to which user ID, use the following:"
-    - "user.info: "
-    - "user.info: [Instructor Only] https://get-a-username-labs-infra.{{ route_subdomain }}/admin"
-    - "user.info: "
-    - "user.info: [Instructor Only] Admin login with 'admin' / '{{ workshop_openshift_user_password }}'"
+  agnosticd_user_info:
+    msg: |
+      URL for attendees to get a username assigned to them and links to labs:
+    
+      https://get-a-username-labs-infra.{{ route_subdomain }}
+    
+      You should share this URL (or a shortlink for it) -- It is all they will need to get started!
+    
+      [Instructor Only] To access the admin see which user is assigned to which user ID, use the following:
+    
+      [Instructor Only] https://get-a-username-labs-infra.{{ route_subdomain }}/admin
+    
+      [Instructor Only] Admin login with 'admin' / '{{ workshop_openshift_user_password }}'
 
 - name: output workshop username distribution URL
-  debug:
-    msg:
-    - "user.info: "
-    - "user.info: Individual module guide URLs (in case you need them for direct linking):"
-    - "user.info: "
-
-
+  agnosticd_user_info:
+    msg: |
+      Individual module guide URLs (in case you need them for direct linking):
+    
 - name: output workshop info guides
   when: ("m1" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 1 http://guides-m1-labs-infra.{{ route_subdomain }}"
+      Module 1 http://guides-m1-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m2" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 2 http://guides-m2-labs-infra.{{ route_subdomain }}"
+      Module 2 http://guides-m2-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m3" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 3 http://guides-m3-labs-infra.{{ route_subdomain }}"
+      Module 3 http://guides-m3-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m4" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 4 http://guides-m4-labs-infra.{{ route_subdomain }}"
+      Module 4 http://guides-m4-labs-infra.{{ route_subdomain }}
```

Comment:
Did you mean to use a yaml block quote here for the `msg:` line?

Reply:
Updated with a pipe